utils/instructionTools.ts

```diff
@@ -44,176 +39,6 @@ export const validateInstruction = async ({
   return isValid
 }
 
-export async function getFriktionDepositInstruction({
```

Comment:
why we want to remove this?

Reply:
It's not used anywhere, they moved their instructions to another file. Just doing some clean up in the codebase.